resource/doc/features.md

```diff
@@ -1,6 +1,26 @@
 # Features
 
-Features are individual inputs that (VMAF) models may use to fuse into a final quality metric score. Note that an existing quality metric can serve as a feature of a new quality metric. Features can also be enabled individually without fusing (e.g., when using VMAF via FFmpeg)."
+Features are individual inputs that (VMAF) models may use to fuse into a final quality metric score. Note that an existing quality metric can serve as a feature of a new quality metric. Features can also be enabled individually without fusing (e.g., when using VMAF via FFmpeg).
+
+## Feature Overview
+
+This is an overview of the available features including their identifier (e.g., when using libvmaf through `ffmpeg`) and the individual metrics that will be computed as part of the feature:
+
+| Feature name      | Identifier      | Core feature? | Individual Metrics                                             |
+|-------------------|-----------------|---------------|----------------------------------------------------------------|
+| VIF               | `vif`           | Yes           | `vif_scale0`, `vif_scale1`, `vif_scale2`, `vif_scale3`         |
+| Motion2           | `motion`        | Yes           | `motion`, `motion2`                                            |
+| ADM               | `adm`           | Yes           | `adm2`, `adm_scale0`, `adm_scale1`, `adm_scale2`, `adm_scale3` |
+| [CAMBI](cambi.md) | `cambi`         | No            | `cambi`                                                        |
+| CIEDE2000         | `ciede`         | No            | `ciede2000`                                                    |
+| MS-SSIM           | `float_ms_ssim` | No            |                                                                |
+| PSNR              | `psnr`          | No            | `psnr_y`, `psnr_cb`, `psnr_cr`                                 |
+| PSNR-HVS          | `psnr_hvs`      | No            | `psnr_hvs`, `psnr_hvs_y`, `psnr_hvs_cb`, `psnr_hvs_cr`         |
+| SSIM              | `float_ssim`    | No            |                                                                |
+
+**Note:** Depending on the build of libvmaf, not all features may be available.
```

Comment:
@kylophone Included this note. For me that would already suffice in case I got an error about some feature not being present. (I know there are more features than those, but they all seem to be available in the ffmpeg builds that I have.)